### PR TITLE
feat(config): import device configs from Z-Wave Alliance (part 1: no params)

### PR DESCRIPTION
**Pull request by blhoward2**
This is the first push of zWave Alliance files, which have no parameters included. Only minor naming changes to present files are included alongside new devices. In support of #1349

### CODE REVIEW

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x0131/zt_zweuzbee.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// Zipatile
+{
+	"manufacturer": "Zipato",
+	"manufacturerId": "0x0131",
+	"label": "ZT.ZWEUZBEE",
+	"description": "Zipatile",
+	"devices": [
+		{
+			"productType": "0x0x03",
+			"productId": "0x0x01",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/zt_zweuzbee.json]: That is not a valid HEX ID

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Please remove `\r` and `\t` from the descriptions

**Comment by AlCalzone** [comment]
@blhoward2

Do we need to check before merging whether the label of a device is the start of another one? Example: `packages\config\config\devices\0x001d\vrc0p_v1.json` vs. `packages\config\config\devices\0x001d\vrc0p.json`. Or are these two different devices?

How did you determine the file labels? Or didn't you change them? Example: `packages\config\config\devices\0x0157\gr-105n.json` vs its label

**Comment by blhoward2** [comment]
I may have missed that one. I’m having to do these ones more manually since they have no parameters to use to distinguish from each other. Where the label doesn’t already match I do the best I can by dropping some last characters/suffixes and comparing the files to each other. If they match it combines them and keeps the shortened label (e.g. XXXXXUS and XXXXXEU become XXXXX) and then re-adding the suffix if that didn’t result in them being merged. There are limits though as to how aggressive I can be or we add different models together. That said, I can probably add v1 to that suffix list without an issue. I’ve been using my review of this set to train it a little better if you will. 

There are a few with very similar file names but which are actually different devices according to the descriptions. 

If a device existed the file name stays the same unless I manually change it. If it’s new it uses the label as it did before.

**Comment by blhoward2** [comment]
> @blhoward2
> 
> Do we need to check before merging whether the label of a device is the start of another one? Example: `packages\config\config\devices\0x001d\vrc0p_v1.json` vs. `packages\config\config\devices\0x001d\vrc0p.json`. Or are these two different devices?
> 
> How did you determine the file labels? Or didn't you change them? Example: `packages\config\config\devices\0x0157\gr-105n.json` vs its label

These are two different devices. One is an in-wall outlet, one is an appliance module.

File: packages/config/config/devices/0x0131/zt_zweuzbee.json (hunk 1 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0131/zt_zweuzbee.json]: Fixed

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
@AlCalzone Unless you see other errors, these should be ready to merge. I fixed everything you noted, plus some. These should be all of the no-parameter devices.

Note: There are some places where there are two spaces and it should be one. Rather than rebuild these, that is a simple grep/sed on the command line. Doing so actually fixes a bunch of our current files too, though, so it was going to pull those into this pile making it hard for you to review. **I'll submit a second PR fixing the 2-->1 space across all current files and these ones once these are merged.** I'll of course fix importConfig.ts too, but I'd already made a bunch of manual changes to these so I don't want to have to redo all of that.

File: packages/config/config/devices/0x0002/hrvccm.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Danfoss HRVCCM
+// Danfoss H
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/hrvccm.json]: wonder what happened here 🤷🏻‍♂️

File: packages/config/config/devices/0x007a/506219.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
 	},
 	"paramInformation": {
 		"0": {
-			"label": "Switching/dimming/shutters dual-surface:",
+			"label": "Switching/dimming/shutters dual-surface: ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x007a/506219.json]: The old label had the same flaw, but we should make sure that there are no trailing punctuation or whitespaces in labels

File: packages/config/config/devices/0x008a/molite.json [hunk 4]
```diff
@@ -88,7 +88,7 @@
 					"value": 1
 				},
 				{
-					"label": "Mode 2: normal operation mode:",
+					"label": "Mode 2: normal operation mode: ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x008a/molite.json]: dito

File: packages/config/config/devices/0x014f/ps15emz51.json [hunk 5]
```diff
@@ -40,7 +40,7 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED ON when the load is ON,",
+					"label": "LED ON when the load is ON, ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x014f/ps15emz51.json]: dito

File: packages/config/config/devices/0x015d/zen22.json [hunk 6]
```diff
@@ -29,7 +29,7 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Upper paddle turns the light on,",
+					"label": "Upper paddle turns the light on, ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015d/zen22.json]: dito

File: packages/config/config/devices/0x016a/ft132.json [hunk 7]
```diff
@@ -160,8 +160,8 @@
 			]
 		},
 		"90": {
-			"label": "Enables/disables parameter 91 and 92 below:",
-			"description": "Enables/disables parameter 91 and 92 below:",
+			"label": "Enables/disables parameter 91 and 92 below: ",
+			"description": "Enables/disables parameter 91 and 92 below: ",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x016a/ft132.json]: dito

File: packages/config/config/devices/0x031e/lzw36.json [hunk 8]
```diff
@@ -47,7 +47,7 @@
 	"paramInformation": {
 		"1": {
 			"label": "Light Dimming Speed",
-			"description": "This changes the speed in which the attached light dims up or down. A setting of 0 should turn the light immediately on or off (almost like an on/off switch). Increasing the value should slow down the transition speed.\n      Range:0-98\n      Default: 4",
+			"description": "This changes the speed in which the attached light dims up or down. A setting of 0 should turn the light immediately on or off (almost like an on/off switch). Increasing the value should slow down the transition speed. Range:0-98 Default: 4",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json:50]: 👍🏻 These are nice!

File: packages/config/config/devices/0x0060/ad142.json [hunk 9]
```diff
@@ -1,18 +0,0 @@
-// Everspring AD142
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/ad142.json]: What happened here? Merged with another one?

File: packages/config/config/devices/0x0060/an157.json [hunk 10]
```diff
@@ -1,22 +0,0 @@
-// Everspring AN157
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0060/an157.json:1]: And here?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I have a few nits regarding label sanitization, but mostly looks good.

File: packages/config/config/devices/0x0002/hrvccm.json (hunk 2 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0002/hrvccm.json]: Looks like I just missed updating the comment when either merging or making the identifier more descriptive. Fixed.

File: packages/config/config/devices/0x008a/molite.json (hunk 4 above)
Comment (blhoward2) [reply packages/config/config/devices/0x008a/molite.json]: Fixed across all files

File: packages/config/config/devices/0x015d/zen22.json (hunk 6 above)
Comment (blhoward2) [reply packages/config/config/devices/0x015d/zen22.json]: Fixed across all files

File: packages/config/config/devices/0x016a/ft132.json (hunk 7 above)
Comment (blhoward2) [reply packages/config/config/devices/0x016a/ft132.json]: Fixed across all files

File: packages/config/config/devices/0x014f/ps15emz51.json (hunk 5 above)
Comment (blhoward2) [reply packages/config/config/devices/0x014f/ps15emz51.json]: Fixed globally

File: packages/config/config/devices/0x0060/ad142.json (hunk 9 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0060/ad142.json]: Not sure but I re-added it. Had to have been my manual mistake as this runs non-destructively (at least for now). I think I meant to delete an142 when I was merging devices.

File: packages/config/config/devices/0x0060/an157.json (hunk 10 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0060/an157.json:1]: This one is correct. It was merged with an145 (and an148/an142). They are actually slightly different devices (screw in lamp holder and plug-in lamp modules), but the manufacturer used the same product type/id for all.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**